source/features/tab-size.css

```diff
@@ -1,41 +1,30 @@
-/* Make tab-indented code more readable on GitHub and Gist */
+/*
+GitHub doesn’t apply the user’s "tab size" setting in a few places, leaving the 8-character default.
+This feature reduces it to 4 characters, while allowing GitHub’s own tab size setting to override it in its children.
 
-* {
-	tab-size: var(--tab-size) !important;
-}
+There’s also one place where GitHub incorrectly uses the "user configuration" attribute to set it to 8, so we’re explicitly overriding that. #4833
 
-[data-tab-size='2'] {
-	--tab-size: 2;
-}
+--tab-size is used to allow the user to override it and to be picked up by `show-whitespace`
+*/
 
-:root, /* Changes the browser default */
-.comment-body [data-tab-size='8'], /* User setting is ignored in comments #4833 */
-[data-tab-size='4'] {
+:root,
+.tab-size[data-tab-size='8'] {
 	--tab-size: 4;
+	tab-size: var(--tab-size);
 }
```

Comment:
The `tab-size` feature overrides the tab size. This one single selector overrides it. I think this is the most consistent situation we got so far

Reply:
Well, you could have just reveted [`eb6f729` (#6480)](https://github.com/refined-github/refined-github/pull/6480/commits/eb6f72935f3cf62db5c3917044fe3e2c153d6a41)

Now this selector isn't specific enough🤔

Reply:
Heh you're right I didn't even notice that I just undid your commit 😅

Reply:
It's fine, I guess I'm just bad at expressing my intent (and tend to complicate things) 😂